crates/binjs_io/src/entropy/mod.rs

```diff
@@ -76,6 +79,16 @@ pub struct Options {
     /// from the compressed files. Not yet used for (de)compression
     /// at this stage.
     string_tables: KindedStringMap<SymbolInfo>,
+
+    /// Statistics obtained while writing.
+    /// If several files are written with the same options, we accumulate
+    /// statistics.
+    content_lengths: Rc<RefCell<write::ContentInfo<usize>>>,
+}
+impl Options {
+    pub fn write_statistics(&self) -> Ref<write::ContentInfo<usize>> {
```

Comment:
this sounds like this method writes
can it be renamed to `statistics_for_write` or something?